libs/eo/core/shell/jest.config.js

```diff
@@ -25,6 +25,13 @@ module.exports = {
     },
   },
   coverageDirectory: '../../../../coverage/libs/eo/core/shell',
+  /**
+   * https://stackoverflow.com/questions/42260218/jest-setup-syntaxerror-unexpected-token-export
+   * NG2-charts has an error with lodash-es, Jest tests fail if this is not added.
+   */
+  moduleNameMapper: {
+    'lodash-es': 'lodash',
+  },
```

Comment:
suggestion: Try moving this to the root `jest.config.js` to verify whether project-specific `jest.config.js` configurations extend from it. Otherwise, we probably have to add it to all libraries that depend on a library using `ng2-charts`.

Reply:
@LayZeeDK, @Klemmensen and i actually had a talk about this, and it might be better for us to move to ngx-chart instead in the future. Lots of people have had problems in their projects when using ng2-charts, because it has a dependency with lodash-es. NGX-chart does not.

Reply:
If we can test "as is", with ngx charts, it is also a benefit. Some exploration could be done here :-D

Reply:
Well, we should also use `lodash-es` over `lodash` to keep a minimal bundle size impact.

Another major difference is that `@swimlane/ngx-charts` renders D3 SVG charts whereas `ng2-charts` renders a canvas element.

In favor of SVG:
- Higher testability as we can interact with SVG elements in our integration and end-to-end tests
- More style customizations as we can apply CSS to SVGs
- Possibly OK for accessibility given enough control of the SVG structure or if this is taken into the account by the chart library

In favor of canvas:
- Canvas performance cannot be beat using SVG. With many charts on one page or many data points in the same chart, performance drops or grinds to a halt when using SVG-based charts for the same reason as having tens of thousands of HTML elements on the page does.

Reply:
Moved the moduleNameMapper to project level, and removed it from specific areas inside the project.

We will look at NG2 vs NGX in the future.